manifests/plugin/remote_execution/ssh.pp

```diff
@@ -53,11 +53,9 @@
     listen_on => $listen_on,
   }
 
-  if $ssh_kerberos_auth {
```

Comment:
This parameter is now unused. We should drop it.

Reply:
It's still used in `templates/plugin/remote_execution_ssh.yml.erb` and from what I understood this is still a valid configuration, only no additional package is required?

Reply:
That's correct. Now it just controls whether `gssapi-with-mic` is added as an authentication method to ssh's `PreferredAuthentications` list

Reply:
Ah my bad.